modules/n1ql/pages/n1ql-language-reference/reservedwords.adoc

```diff
@@ -193,83 +207,97 @@ The following keywords are reserved and cannot be used as unescaped identifiers:
 | POOL
 | xref:n1ql-language-reference/window.adoc#window-frame-extent[PRECEDING]
 | PREPARE
+| PREV
+| PREVIEW
+
+| PREVVAL
 | PRIMARY
 | PRIVATE
-
 | PRIVILEGE
 | xref:n1ql-language-reference/join.adoc#use-hash-hint[PROBE]
 | PROCEDURE
+
 | PUBLIC
 | xref:n1ql-language-reference/window.adoc#window-frame-clause[RANGE]
 | RAW
-
 | REALM
 | REDUCE
 | RENAME
+
+| REPLACE
 | xref:n1ql-language-reference/window.adoc#nulls-treatment[RESPECT]
+| RESTART
+| RESTRICT
+| READ
+| RECURSIVE
```

Comment:
Please move `READ` between RAW and REALM, and move `RECURSIVE` between REALM and REDUCE.